docker-app/qfieldcloud/core/templates/account/password_reset_from_key.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'admin/base_site.html' %}
+{% load i18n static %}
+{% load bootstrap4 %}
+
+{% block title %}
+  {% trans 'Change Password' %}
+{% endblock %}
+
+{% block content %}
+  <div class="card">
+    <div class="card-body">
+
+      <h5 class="card-title mb-3">
+        {% if token_fail %}
+          {% trans 'Bad Token' %}
+        {% else %}
+          {% trans 'Change Password' %}
+        {% endif %}
+      </h5>
+
+      {% if token_fail %}
+
+        <p class="card-text">
+          {% url 'account_reset_password' as passwd_reset_url %}
+          {% blocktrans %}The password reset link was invalid, possibly because it has already been used. Please request a <a href="{{ passwd_reset_url }}">new password reset</a>.{% endblocktrans %}
```

Comment:
```suggestion
          {% blocktrans %}
            The password reset link was invalid, possibly because it has already been used. Please request a <a href="{{ passwd_reset_url }}">new password reset</a>.
          {% endblocktrans %}
```

Can you try to make `blocktrans` on it's own individual lines. Most of the templates are following this convention.

There are a few more in this PR that need to be reindented.

Reply:
Thanks for spotting this, indeed, I might need to change my IDE template formatter :-)